spec/parsers/pypi_spec.rb

```diff
@@ -204,6 +204,35 @@
     })
   end
 
+  it "parses dependencies from pip-resolved-dependencies.txt, with tree" do
+    expect(described_class.analyse_contents("pip-resolved-dependencies.txt", load_fixture("pip_with_tree/pip-resolved-dependencies.txt"))).to eq({
+      platform: "pypi",
+      path: "pip-resolved-dependencies.txt",
+      dependencies: [
+        { name: "black", requirement: "==23.12.0", type: "runtime" },
+        { name: "aiohttp", requirement: "3.9.5", type: "runtime" },
```

Comment:
note how the regular requiremens.txt-compatible lines are parsed with "==" version range prefix, but the tree-syntax lines do not get that, because they're parsed from a string like `"├── aiohttp [required: >=3.7.4, installed: 3.9.5]"`. 

we could theoretically prefix them with "==" ourselves for consistency, but it doesn't seem like there's a big need since they both mean the same thing.

Reply:
this comment is irrelevant now that I'm using the JSON instead of requirements-compatible file